index.js

```diff
@@ -37,12 +37,20 @@ function record(voiceConnection) {
     voiceDataStream.on("error", console.error);
 }
 
+function collectionValuesContain(c, o) {
```

Comment:
This function is obsolete due to my other change.

Reply:
I will redact it as it seems utterly useless.